src/zenml/cli/stack_components.py

```diff
@@ -288,6 +305,14 @@ def register_stack_component_command(
             )
             print_model_url(get_component_url(component))
 
+        connect_stack_component_with_service_connector(
+            component_type=component_type,
+            name_id_or_prefix=name,
+            connector=connector,
+            interactive=False,
+            no_verify=False,
+        )
+
```

Comment:
You also need a resource ID argument for some resource types, such as cloud Kubernetes clusters. You're also missing out on the best part of linking a component to a connector: the interactive mode.